src/binary-reader.cc

```diff
@@ -466,6 +466,17 @@ static void write_indent(LoggingContext* ctx) {
     LOGF_NOINDENT(__VA_ARGS__); \
   } while (0)
 
+static void logging_on_error(BinaryReaderContext* context,
```

Comment:
This was a bug, basically the reader would crash when you turn on logging and there is an error.